app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -420,30 +386,6 @@ private String getFormattedPcrString(double pcrValue) {
         }
 
         @Override public void updatePatientConditionUi(String generalConditionUuid) {
```

Comment:
Acknowledged. I'm inclined to track this with an issue instead of just by commenting out the relevant code, given that reimplementation would be completely different in the future. Thanks for the information!